engines/order_management/spec/services/order_management/stock/estimator_spec.rb

```diff
@@ -45,22 +45,22 @@ module Stock
           it "does not return shipping rates from a shipping method" do
             allow_any_instance_of(Spree::ShippingMethod).
               to receive_message_chain(:calculator, :available?).and_return(false)
-            shipping_rates = subject.shipping_rates(package)
+            shipping_rates = subject.shipping_rates(package: package)
             expect(shipping_rates).to eq []
           end
         end
 
         context "the currency matches the order's currency" do
           it "returns shipping rates from a shipping method" do
-            shipping_rates = subject.shipping_rates(package)
+            shipping_rates = subject.shipping_rates(package: package)
```

Comment:
I am not a big fan of keyword arguments, specially when they are mandatory because then we have lots of this repetitions package: package.
I'd make package a normal argument.
This is just my opinion, please leave it if you like it.

Reply:
No that's a really good point about keyword arguments; I only like to use them if they're really necessary, and I think I was just on autopilot here :) For the other methods, I added them because I don't like seeing code like 

`shipping_methods(distributor, true, nil, false)` 

since it's hard to know what those parameters mean if they're not named. So that's why I used them in `DistributorShippingMethods` but you're correct it's unnecessary here, I got carried away :)

Reply:
Oh, sorry - I took a look at the method definition, and realized why I think named params are good here. The method definition looks like: 

`def shipping_rates(package:, checkout: true, apply_tags: true)`

so I think we should keep it using named arguments to make it clear in the case when you're not using the defaults for `checkout` and `apply_tags`.

Reply:
Ah, it turns out you can mix argument types! So we can have both.

Reply:
81e2f5a4c8838a0ebfb89c651aa5e36d2b5972aa